src/ResourceManager/Resources/Commands.Resources.Test/Providers/GetAzureProviderCmdletTests.cs

```diff
@@ -153,7 +153,7 @@ public void GetsResourceProviderTests()
             };
             var pagableLocations = new Page<Location>();
             pagableLocations.SetItemValue<Location>(locationList);
-            var locationsResult = new AzureOperationResponse<IPage<Location>>()
+            var locationsResult = new AzureOperationResponse<IEnumerable<Location>>()
```

Comment:
@TianoMS @markcowl That is correct. The list subscription locations was never pageable to begin with, it was incorrectly added. The above change to swagger spec @TianoMS referenced fixed the issue.